downstairs/src/main.rs

```diff
@@ -182,6 +182,45 @@ enum Args {
         bind_addr: SocketAddr,
     },
     Version,
+    /*
+    Measure an isolated downstairs
+    */
+    Dynamometer {
+        #[clap(long, default_value = "512", action)]
+        block_size: u64,
+
+        #[clap(short, long, name = "DIRECTORY", action)]
+        data: PathBuf,
+
+        #[clap(long, default_value = "100", action)]
+        extent_size: u64,
+
+        #[clap(long, default_value = "15", action)]
+        extent_count: u64,
+
+        #[clap(long, action)]
+        encrypted: bool,
+
+        // Number of writes to submit at one time to region_write
```

Comment:
If you make these into doc comments (`///`) instead of regular comments, they will show up in the CLI with the `--help` option:
```
Options:
      --block-size <BLOCK_SIZE>
          [default: 512]
  -d, --data <DIRECTORY>

      --extent-size <EXTENT_SIZE>
          [default: 100]
      --extent-count <EXTENT_COUNT>
          [default: 15]
      --encrypted

  -n, --num-writes <NUM_WRITES>
          Number of writes to submit at one time to region_write [default: 1]
```

Reply:
nice, 3cb447e